### ISSUE TITLE
Not able to see Xcode source editor in entention.

### ISSUE THREAD

**Issue by VikasPrajapati27** [qfish/XAlign#96]
i m not able to person step 1 Enable X Align which you explained

**Comment by arthankamal** [comment]
Yes, it's happening for me in Xcode 8.1. Any solutions..?

**Comment by qfish** [comment]
What's your macOS version, and have you run the `XAlign.app` ?

**Comment by VikasPrajapati27** [comment]
my mac os version is 10.11.6,
and yes i have run XAlign.app.

**Comment by arthankamal** [comment]
I faced the same issue, but it's fixed after installing mac OS Sierra

**Comment by AmitBhavsarIphone** [comment]
Yes @VikasPrajapati27  I faced same issue before, I installed mac OS Sierra.
Update your OS.

**Comment by Easyzhan** [comment]
10.11.5也是同样的，same issue on 10.11.5

**Comment by arthankamal** [comment]
@Easyzhan , you need to install 10.12 (mac OS Sierra).

**Comment by Easyzhan** [comment]
@arthankamal i don't want install 10.12,mac run slowly

**Comment by DAKarp** [comment]
Doesn't show up for me, either - running Xcode 8.2.1 on Sierra 10.12.2
